employee.rb

```diff
@@ -0,0 +1,28 @@
+def get_user_input
+  gets.chomp
+end
+
+
+class Employee
+  attr_accessor :name, :salary, :department, :reviews, :evaluation, :give_raise
```

Comment:
You make an attr_accessor here for `:give_raise`, but I think that's not going to do anything here since you /actually/ write out a give_raise method below.